bedrock/base/templates/macros.html

```diff
@@ -301,6 +301,28 @@ <h3>{{ _('Your download link was sent.') }}</h3>
   </section>
 {%- endmacro %}
 
+{% macro fxa_link_fragment(entrypoint, action='signin', utm_params={}) -%}
```

Comment:
This is turning into quite a lot of complex logic for a macro to be responsible for. My gut says we should probably convert this into a jinja helper in Python, so that it can have tests. Thoughts, @pmac ?

Reply:
We can address this in a followup PR.